origins_form_descriptions/origins_form_descriptions.module

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @file
+ * Contains origins_form_descriptions.module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_alter().
+ */
+function origins_form_descriptions_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $route_name = \Drupal::routeMatch()->getRouteName();
+
+  // Add JS/CSS to entity add/edit and canonical forms.
+  if (preg_match('/^entity.\w+.edit_form$/', $route_name)
```

Comment:
Did you try hook_form_BASE_FORM_ID_alter to deal with the base entity forms?

Reply:
Good shout, looks much nicer with this.